roles/artemis/defaults/main.yml

```diff
@@ -68,7 +68,11 @@ artemis_war_url: "https://github.com/ls1intum/Artemis/releases/download/{{ artem
 
 artemis_system_ram_proportion: 0.8  # How much % of the system RAM to allocate to the Artemis service (Default here: 80%)
 
+artemis_database_schema: mysql
```

Comment:
Please add a few lines of documentation here. Especially to the type and platform defaults